src/main/java/org/myrobotlab/service/Python.java

```diff
@@ -1015,14 +1015,14 @@ public List<String> getScriptList() throws IOException {
    * @throws IOException
    */
   public void addScript(String scriptName, String code) throws IOException {
-    PythonConfig c = (PythonConfig)config;
+//    PythonConfig c = (PythonConfig)config;
 
-    File script = new File(c.scriptRootDir + fs + scriptName);
+//    File script = new File(c.scriptRootDir + fs + scriptName);
 
-    if (script.exists()) {
-      error("script %s already exists", scriptName);
-      return;
-    }
+//    if (script.exists()) {
+//      error("script %s already exists", scriptName);
+//      return;
+//    }

```

Comment:
Is all of this meant to be commented out? Can we remove it?

Reply:
removed